unit-tests/func/hw-errors/hw-errors-common.h

```diff
@@ -0,0 +1,99 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../test.h"
+#include <concurrency.h>
+
+using namespace rs2;
+
+const uint8_t l500_trigger_error_opcode = 0x5E;
+
+// Possible error values
+const std::map< uint8_t, std::string > l500_error_report
```

Comment:
If it's `common`, why put l500-specific code here?

Reply:
put in on l500 directory